scalability_and_performance/enabling-workload-partitioning.adoc

```diff
@@ -0,0 +1,74 @@
+:_content-type: ASSEMBLY
+[id="enabling-workload-partitioning"]
+= Workload partitioning
+include::_attributes/common-attributes.adoc[]
+:context: enabling-workload-partitioning
+
+toc::[]
+
+In resource-constrained environments, you can use workload partitioning to isolate {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs.
+
+The minimum number of reserved CPUs required for the cluster management is four CPU Hyper-Threads (HTs).
+With workload partitioning, you annotate the set of cluster management pods and a set of typical add-on Operators for inclusion in the cluster management workload partition.
+These pods operate normally within the minimum size CPU configuration.
+Additional Operators or workloads outside of the set of minimum cluster management pods require additional CPUs to be added to the workload partition.
+
+Workload partitioning isolates user workloads from platform workloads using standard Kubernetes scheduling capabilities.
+
+The following changes are required for workload partitioning:
+
+. In the `install-config.yaml` file, add the additional field: `cpuPartitioningMode`.
++
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: devcluster.openshift.com
+cpuPartitioningMode: AllNodes <1>
+Compute:
```

Comment:
@slovern, everywhere else in the docs this is: `compute:`, should this be uppercase first here instead?

Reply:
thanks @jboxman-rh - well spotted. I've made that change now